app/prover/[teamId]/columns.tsx

```diff
@@ -23,6 +23,7 @@ import { prettyMs } from "@/lib/time"
 export const columns: ColumnDef<Proof>[] = [
   // Block (time since)
   {
+    id: "block_number",
```

Comment:
```tsx
          sorting={[{ id: "block_number", desc: true }]}
```

^ The above line from `prover/[teamId]/page.tsx` was looking for a column with this id; adding this fixed the build error